src/main/java/com/ays/assignment/controller/AssignmentController.java

```diff
@@ -39,4 +44,23 @@ public AysResponse<Void> saveAssignment(@RequestBody @Valid AssignmentSaveReques
         assignmentSaveService.saveAssignment(saveRequest);
         return AysResponse.SUCCESS;
     }
+
+
+    /**
+     * Gets a user by ID.
+     * Requires ADMIN authority.
+     *
+     * @param id The ID of the user to retrieve.
+     * @return A response object containing the retrieved user data.
+     */
+    @GetMapping("/assignment/{id}")
+    @PreAuthorize("hasAnyAuthority('USER')")
```

Comment:
Bu endpointe `ADMIN` erişeceği için burayı düzeltmemiz gerekiyor

Reply:
`ADMIN` göre gerekli değişiklikleri yaptım.